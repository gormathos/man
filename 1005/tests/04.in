-10 7 7 -3 -4 10 8 -2 -8
-6 -10 3 -9 -7 -2 9 -10 7
7 1 3 7 10 -8 -3 -4 10
-6 3 4 -8 -8 0 9 -2 -6
8 -2 -4 -5 5 0 9 1 1
7 8 2 2 6 -3 1 3 -2
5 -5 7 7 -7 -7 7 -4 1
