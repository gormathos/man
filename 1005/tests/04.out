7
-10 7 -3 -4 10 8 -2
-6 -10 -9 -7 -2 9 -10
7 1 7 10 -8 -3 -4
-6 3 -8 -8 0 9 -2
8 -2 -5 5 0 9 1
7 8 2 6 -3 1 3
5 -5 7 -7 -7 7 -4
4465940
